scripts/test_reports/report.py

```diff
@@ -704,7 +694,7 @@ def parse_args():
 one file depending on the source of the results.
 
 TCF results expect a directory with multiple files,
-where sanitycheck expect 1 file per configuration.
+Sanitycheck was updated like a TCF, and now it expects a directory with multiple files too.
```

Comment:
I do not think we need to be compatible with tcf, we can define our own format.

Reply:
Do you mean that we can create files with name like sanitycheck-name_of_the_board__commitname_zephyr.xml?